### PR TITLE
Change2

### PR DESCRIPTION
**Pull request by ThanushiyaMary**
changes held

### CODE REVIEW

**Comment by Madhuyayathi** [comment]
ok

**Comment by Madhuyayathi** [comment]
again issue

**Comment by ThanushiyaMary** [comment]
end

File: Lie [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Heaven is like hell
```
Comment (ThanushiyaMary) [inline Lie]: Hello heaven

Comment (Madhuyayathi) [reply Lie]: correct code

**Pull request closed without merging**